packages/ui/form/PhoneInput.tsx

```diff
@@ -18,17 +18,34 @@ export type PhoneInputProps = {
 };
 
 function BasePhoneInput({ name, className = "", onChange, ...rest }: PhoneInputProps) {
+  useEffect(() => {
+    if (!rest.value) {
+      return;
+    }
+    const formattedValue = rest.value.trim().replace(/^\+?/, "+");
+    onChange(formattedValue);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  let countryCode;
+  if (rest.value) {
+    try {
+      countryCode = parsePhoneNumber(rest.value)?.country || "";
+    } catch {}
+  }
+
```

Comment:
This can be removed if we do not want to make US as default country for invalid phone inputs

Reply:
this hooks gets the country based on user ip on prod when deployed on vercel

Reply:
Understood 👍 , that makes sense